### PR TITLE
SRVKS-188: added http2/gRPC caveat

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]:
```suggestion
To do so, you will need to find the ingressgateway's public address and the application's specific host.
```

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]:
```suggestion
helloworld-go   http://helloworld-go.default.example.com   helloworld-go-2rjbt   helloworld-go-2rjbt   True
```

To stick with the example above.

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]:
```suggestion
In this case, the application's host would be:
```

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]:
```suggestion
. The application's host is part of the URL that is surfaced when using `oc get ksvc`, but without the `http://` prefix. For example:
```

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]:
```suggestion
helloworld-go.default.example.com
```

Comment (markusthoemmes) [reply modules/interacting-serverless-apps.adoc]: Maybe even just refer to the example above and state that that's how you get the application's host and that for this example the host is "helloworld-go.default.example.com". Maybe we can then omit having the same procedure twice?

Comment (abrennan89) [reply modules/interacting-serverless-apps.adoc]: I didn't even realise, good call, thanks!

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [inline modules/interacting-serverless-apps.adoc]: Maybe make clear in the note that this only applies to gRPC.

**Review by markusthoemmes — COMMENTED** [review]
SME approved (modulo the last comment)

File: modules/interacting-serverless-apps.adoc
Comment (abrennan89) [reply modules/interacting-serverless-apps.adoc]: I moved the note up so it makes more sense (above the example). Would it make sense just to show the example with these appended though to show the format?

Comment (markusthoemmes) [reply modules/interacting-serverless-apps.adoc]: That indeed makes a lot of sense. The example should then look like this:

```golang
grpc.Dial(
    "a83e86291bcdd11e993af02b7a65e514-33544245.us-east-1.elb.amazonaws.com:80",
    grpc.WithAuthority("helloworld-go.default.example.com:80"),
    grpc.WithInsecure(),
)
```

Comment (abrennan89) [reply modules/interacting-serverless-apps.adoc]: Perfect, thanks Markus! :slightly_smiling_face:

**Comment by mgencur** [comment]
Except that I don't see this article in https://SRVKS-188--ocpdocs.netlify.com/, the code looks good to me.

QE approved

File: modules/interacting-serverless-apps.adoc
Comment (adellape) [inline modules/interacting-serverless-apps.adoc]: s/./:/

File: modules/interacting-serverless-apps.adoc
Comment (adellape) [inline modules/interacting-serverless-apps.adoc]: Should "host" and "domain" be marked up with backticks? Or maybe not since I don't see "domain" explicitly anywhere.

**Review by adellape — COMMENTED** [review]
Just minor questions. :+1:

File: modules/interacting-serverless-apps.adoc
Comment (markusthoemmes) [reply modules/interacting-serverless-apps.adoc]: This can now actually be reworded to something like: "Ensure that you append the respective port (80 by default) to both hosts as shown in the example."

File: modules/interacting-serverless-apps.adoc
Comment (abrennan89) [reply modules/interacting-serverless-apps.adoc]: I think actually this could be a separate module and this could be restructured as an assembly maybe, let me do this first before we merge maybe!

**Review by adellape — APPROVED** [review]
:fireworks:

**Pull request merged**
